goonmisc/goonmisc.py

```diff
@@ -900,3 +900,9 @@ async def close_menu(
     ):
         with contextlib.suppress(discord.NotFound):
             await message.delete()
+
+    @commands.command(aliases=["readme"])
```

Comment:
There's no point in adding the same alias as is the name of the command
```suggestion
    @commands.command()
```